server/src/test/scala/org/http4s/server/middleware/LoggerSpec.scala

```diff
@@ -0,0 +1,64 @@
+package org.http4s
+package server
+package middleware
+
+import org.http4s.dsl._
+
+/**
+  * Common Tests for Logger, RequestLogger, and ResponseLogger
+  */
+class LoggerSpec extends Http4sSpec {
```

Comment:
The log statements could be tested if an [OutputStreamAppender](https://logback.qos.ch/apidocs/ch/qos/logback/core/OutputStreamAppender.html) is configured, though it might not be a great deal of fun.